src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -72,6 +87,14 @@ object VersionInjectorPlugin extends AutoPlugin {
     (gitCommand("describe").!!).trim
   }
 
+  def gitMostRecentCommit(path: File): String = {
+    gitCommand("log", "-1", "--format=%H", path).!!.trim
```

Comment:
I'd move this into the task def below - it's not used elsewhere, so a separate helper isn't really needed. Maybe add a comment to it, though?